help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -257,4 +257,17 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
 ## Additional resources {#additional-resources}
 
 * [!DNL Salesforce Marketing Cloud] [API](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/apis-overview.html)
-* [!DNL Salesforce Marketing Cloud] [documentation](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/updateContacts.html) explaining how contacts are updated with the specified information in the specified attribute groups.
\ No newline at end of file
+* [!DNL Salesforce Marketing Cloud] [documentation](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/updateContacts.html) explaining how contacts are updated with the specified information in the specified attribute groups.
+
+## Changelog
+
+| Release Date | Connector Version | Description |
+| --- | --- | --- |
+| Feb 21, 2023 | 1.0 | **Changed** <ul><li>Documentation: Reference link to [!DNL Salesforce Marketing Cloud Account Engagement] as a mandatory subscription</li></ul> |
+| Feb 17, 2023 | 1.0 | **Changed** <ul><li>Connector: Added the *Expires In* header to the authentication payload as it is supported by [!DNL Salesforce Marketing Cloud].</li> <li>Connector: Updated Pebble template to include a hard-coded *Email Demographics* mapping at the end of the body, `PLATIR-27068`</li></ul> **Fixed** <ul><li>Connector: Corrected domain name regex.</li></ul> |
+| Feb 08, 2023 | 1.0 | **Changed** <ul><li>Documentation: Back-link reference anchor to gather credentials, `PLATIR-25925`</li> <li>Documentation: Corrected Markdown formatting used in the custom domain explanation</li></ul> **Fixed** <ul><li>Connector: Removed extraneous comma from pebble template</li></ul> |
```

Comment:
same comment as above about the Pebble template. The real question is why the comma was removed from the Pebble template. if you're not aware of what exactly this comma removal fixed, we can leave this row out altogether from the docs.